plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -115,7 +115,23 @@ const fetchPluginData = async ({createNode, reporter}) => {
                         type: 'JenkinsPlugin',
                         contentDigest: crypto.createHash('md5').update(`plugin${pluginData.name.trim()}`).digest('hex')
                     }
-                }).then(() => {
+                });
+                if (!p || !p.then) {
```

Comment:
Can `createNode` return something else than a valid promise?

Reply:
It shouldn't. I was going to look into it, but it turned a random null once in a while for me